tools/README.md

```
@@ -1373,6 +1373,29 @@ when the panes it notified read their files.)
 
 ## Conventions worth copying
 
+### ⛔ `gh --jq` IS NOT `jq` — it corrupts lookarounds before jq sees them
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)